portfolio/src/main/webapp/script.js

```diff
@@ -32,3 +32,26 @@ async function getRandomMyNameUsingAsyncAwait() {
   const name = await response.text();
   document.getElementById('name-container').innerText = name;
 }
+
+function getJSONString() {
+  fetch('/data').then(response => response.json()).then((json) => {
+    // stats is an object, not a string, so we have to
+    // reference its fields to create HTML content
+
+    const jsonListElement = document.getElementById('json-string');
+    jsonListElement.innerHTML = '';
+    jsonListElement.appendChild(
+        createListElement(json[0]));
+    jsonListElement.appendChild(
+        createListElement(json[1]));
+    jsonListElement.appendChild(
+        createListElement(json[2]));
```

Comment:
You can do this in a for loop, that way you're not tied to exactly the way it's defined on the server side.

I'd recommend using a "for of" loop: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of

Reply:
Ohh makes sense! Just committed again, should have fixed the issues we talked about :)